.github/workflows/issue-comment-created.yml

```diff
@@ -0,0 +1,16 @@
+name: Issue Comment Created Triage
+
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  issue_comment_triage:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2 
```

Comment:
This was my bad in the migration plan, I don't think this is necessary for this workflow 👍 

```suggestion
```